azkaban-common/src/main/java/azkaban/storage/LocalStorage.java

```diff
@@ -36,34 +37,32 @@
 public class LocalStorage implements Storage {
   private static final Logger log = Logger.getLogger(LocalStorage.class);
 
-  final File baseDirectory;
+  final File rootDirectory;
 
-  public LocalStorage(File baseDirectory) {
-    this.baseDirectory = validateBaseDirectory(createIfDoesNotExist(baseDirectory));
+  @Inject
+  public LocalStorage(AzkabanCommonModuleConfig config) {
```

Comment:
Is going to be a future convention for juice usage in Azkaban? Every class calls `AzkabanCommonModuleConfig ` ?